docs/topics/loaders.rst

```diff
@@ -64,7 +65,11 @@ data that will be assigned to the ``name`` field later.
 Afterwards, similar calls are used for ``price`` and ``stock`` fields
 (the latter using a CSS selector with the :meth:`~ItemLoader.add_css` method),
 and finally the ``last_update`` field is populated directly with a literal value
-(``today``) using a different method: :meth:`~ItemLoader.add_value`.
+(``today``) using a different method: :meth:`~ItemLoader.add_value`. To note 
+:meth:`~ItemLoader.add_css` and :meth:`~ItemLoader.add_xpath` methods will always
+return a list as they are actually calling selector ``getall()`` method over 
```

Comment:
add_css and add_xpath methods don't return anything - I see what you mean, but it seems a different wording is needed.

Reply:
How about: 

> ... methods will always store a list in the resulting item's field as they are actually calling selector ``getall()`` method over ...